app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MediaRanker</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
+  <header class="app-header">
+    <h1><a href="/">Media Ranker</a>
+    <small>Ranking the Best of Everything</small></h1>
+
+    <nav class="app-header-nav">
+      <div class="link-header"><%= link_to 'View top media', works_top_path %></div>
+      <div class="link-header"><%= link_to 'View all media', works_path %></div>
+      <div class="link-header"><%= link_to 'Add a new work', new_work_path %></div>
+      <div class="link-header"><%= link_to 'View all users', users_path %></div>
+      <% if session[:user_id] %>
```

Comment:
It would be more reliable to check `@user` over `session[:user_id]`. You would need to move your UsersController's `find_user` controller filter to the ApplicationController to make it available to this view. However, this view is more closely related to the controller, and it's easier to read that way.

By checking `session[:user_id]` directly, and not checking `@user`, we risk two things:

1. `session[:user_id]` is very close to implementation, and isn't descriptive; if I didn't work on this project, I'd be unsure what `session[:user_id]` meant.
2. This makes this code very brittle... This means it's hard to change and update. We What if we decide to change how we log in, and now we use `session[:current_user_id]` or something else instead? It would be easier to update the `find_user` controller filter once, compared to finding every instance of `session[:user_id]` and updating that.